composer.json

```diff
@@ -9,7 +9,7 @@
   "require": {
     "php": ">=7.1",
     "ext-sockets": "*",
-    "apache/thrift": "0.11.*",
+    "apache/thrift": ">=0.11, <0.16",
```

Comment:
Allowed only known (for now) versions, which is safe to use (based on investigated changelog)

Reply:
As an alternative - I can narrow down supported versions only to `0.15.*`, if you wish.